lumicks/pylake/population/tests/generate_trace_data.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
```

Comment:
Last optional remark: consider moving this file and the data into a subfolder together to clearly separate them from the regular tests files.

Before:
```
tests/generate_trace_data.py
tests/trace_data.npy
```

After:
```
tests/data/generate_trace.py
tests/data/trace.npy
```